src/script/Config.ts

```diff
@@ -84,7 +84,7 @@ const config = {
   ALLOWED_IMAGE_TYPES: ['image/bmp', 'image/gif', 'image/jpeg', 'image/jpg', 'image/png', 'image/webp'],
 
   /** Which min and max version of the backend api do we support */
-  SUPPORTED_API_RANGE: [1, env.ENABLE_DEV_BACKEND_API ? Infinity : 8],
+  SUPPORTED_API_RANGE: [8, env.ENABLE_DEV_BACKEND_API ? Infinity : 8],
```

Comment:
Can we use a shared min variable instead of a magic number? We have the hardcoded 8 now in several different places